### PR TITLE
DSY-1667 docs: create icons doc

### PR DESCRIPTION
**Pull request by lauralucca**
DSY-1667

### Proposed changes

This PR brings the following changes:

- Doc file about how to use icons

### Related issue

[DSY-1667 - Criar Documentacão sobre utilização Ícones no ZH](https://natura.atlassian.net/browse/DSY-1667)

### Dependencies added/removed (if applicable)

No dependencies were added or removed for this change.

---

### Testing

N/A

#### How to test

N/A

#### Test configuration

N/A

---

## Checklist

- [ ] I have added corresponding labels to this PR (like bug, enhancement...);
- [ ] My commits follow the [Conventional Commits 1.0 Guidelines](https://www.conventionalcommits.org/en/v1.0.0/);
- [ ] My code follows the style guidelines of this project;
- [ ] I have performed a self-review of my own code;
- [ ] I have mapped technical debts found on my changes;
- [ ] I have made corresponding changes to the documentation (if applicable);
- [ ] My changes generate no new warnings or errors;

### CODE REVIEW

**Review by arielwb — CHANGES_REQUESTED** [review]

File: packages/web/docs/icons.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- [Icons](https://zeroheight.com/28db352be/p/94367e-icon-library/b/6154b9) and [Icon Names](https://github.com/natura-cosmeticos/natds-commons/blob/master/packages/natds-icons/dist/natds-icons.json)
+
+Here is a quick example to get you started:
+```javascript
+import React from 'react';
+import { Icon } from '@naturacosmeticos/natds-web';
+
+export default Component = () => (
+  <Icon
+    name="filled-default-mockup"
```
[the remark this answers is not in the document]
Comment (lauralucca) [reply packages/web/docs/icons.md:47]: Escolhi o que achei mais fofo ☺️

File: packages/web/docs/icons.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    size="standard"
+    color="default"
+  />
+);
+```
+
+> For component detailed API and usage examples, check out the [docs](https://natds-web.prd.naturacloud.com/) of the component you are using.
```
[the remark this answers is not in the document]
Comment (lauralucca) [reply packages/web/docs/icons.md]: Feito!

File: packages/web/docs/icons.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+---
+
+### Not a React-based project?
+
+You can also install icons separately. Please check `natds-icons` [docs](./packages/icons/README.md) for more info.
```
[the remark this answers is not in the document]
Comment (lauralucca) [reply packages/web/docs/icons.md]: Nossa, super tinha pensado isso e esqueci! Valeusss!

**Review by Design-System-Natura — APPROVED** [review]
LGTM ⭐

**Review by yasminnvaz — APPROVED** [review]
LGTM ⭐

**Pull request merged**